tw-tasks-core/src/main/java/com/transferwise/tasks/cleaning/TasksCleaner.java

```diff
@@ -47,6 +48,10 @@ public class TasksCleaner implements ITasksCleaner, GracefulShutdownStrategy, In
 
   @Override
   public void afterPropertiesSet() {
+    if (tasksProperties.getStopTasksCleaner()) {
+      log.info("Tasks cleaner is disabled.");
+      return;
+    }
```

Comment:
Think the check should be in `applicationStarted`, this function just initialises the class

https://github.com/transferwise/tw-tasks-executor/blob/master/tw-tasks-core/src/main/java/com/transferwise/tasks/cleaning/TasksCleaner.java#L138-L141